cli_reference/basic_cli_operations.adoc

```diff
@@ -327,7 +327,11 @@ work for other object types.
 |`exec`
 |`oc exec -p _<pod_ID>_ \`
 `-c _<container_ID>_ -- _<command>_`
-|Execute a command in a already-running container.
+|Execute a command in a already-running container. It will default to the first container if none is specified.
+
+|`rsh`
+|`oc rsh _<pod_ID>_ `
```

Comment:
The space before the closing backtick is breaking the rendered markup for some reason. :/